tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/project.json

```diff
@@ -26,7 +26,7 @@
   "tools": {
     "Microsoft.AspNetCore.Server.IISIntegration.Tools": {
       "version": "1.0.0-preview1-final",
-      "imports": "net451"
+      "imports": "portable-net45+win8+dnxcore50"
```

Comment:
This has to be updated to fix the continuous deployment issue in Azure as well as from other laptops